Documentation/capabilities-guide.md

```diff
@@ -222,6 +223,101 @@ root
 nc: bind: Permission denied
 ```
 
+## Overriding capabilities
+
+Capabilities sets are typically defined when creating images, as they are tightly
+linked to specific app requirements. However, image consumers may need to further
+tweak/restrict the set of available capabilities in specific local scenarios.
+This can be done either by permanently patching the manifest of specific images, or
+by overriding capability isolators via command line arguments.
+
+### Patching images
+
+Image manifests can be manipulated manually, by unpacking the image and editing
+the manifest file, or via helper tools like `actool`.
+In order to override pre-defined capabilities set, it is enough to replace any
+existing capability isolators with a new custom one.
+
+`actool patch-manifest` provides two options to manipulate the retain set
```

Comment:
The `patch-manifest` subcommand to `actool` manipulates the capabilities sets defined in an image. `actool patch-manifest --capability` changes the `retain` capabilities set. `actool patch-manifest --revoke-capability` changes the `remove` set. These commands take an input image, modify its existing capabilities sets, and write the changes to an output image, as shown in this example: